ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```
@@ -2209,6 +2210,15 @@ objects:
         openshift-install --dir=${ARTIFACT_DIR}/installer wait-for install-complete 2>&1 | grep --line-buffered -v password &
         wait "$!"
 
+        END_TIME=$(echo "$(date +%s)")
+
+        function update_config_map() {
+          echo "Updating Cluster Infra ConfigMap with the start and end times."
+          oc patch configmap openshift-install -n openshift-config -p "{'data':{'installTimes':{'startTime':'$1','endTime':'$2'}}}"
+        }
+
+        update_config_map $START_TIME $END_TIME
```

We don't need to use a function here.
```suggestion
        echo "Updating Cluster Infra ConfigMap with the start and end times."
        oc patch configmap openshift-install -n openshift-config -p "{'data':{'installTimes':{'startTime':'$START_TIME','endTime':'$END_TIME'}}}"
```